src/views/tickets/Yara.js

```diff
@@ -45,6 +45,22 @@ export default class Yara extends Component {
     return yara;
   }
 
+  getRuleName = rule => {
+    console.log(rule);
+    if (rule.metas) {
+      for (const item of rule.metas) {
+        console.log(item);
+        if (item.id === 'desc') {
```

Comment:
Since both these branches return the same value, you can collapse both cases using `||` into the first conditional check, and remove the second branch.